R/report-utils-xls.R

```diff
@@ -439,7 +440,15 @@ write.xls.report <- function(properties.env,report.env,file="isobar-analysis.xls
       x <- x[x[,'end.pos'] <= protein.length,]
       if (nrow(x) == 0) return(c(seq.cov=NA))
       seqq <- rep(FALSE,protein.length);
-      for (i in seq_along(nrow(x))) seqq[x[i,'start.pos']:x[i,'end.pos']] <-  TRUE
+      for (i in seq_along(nrow(x))) {
```

Comment:
Both for performance and correctness it's better to remove those peptides from `x` before entering the loop -- at the same line, where `x[,'end.pos'] <= protein.length` is being checked (actually, `end.pos` should be checked for NA before that one check). If there are no valid peptides it's better to have `seq.cov=NA` than `seq.cov=0`.